### PR TITLE
Add BIRLS columns to form526_submissions

### PR DESCRIPTION
**Pull request by edmangimelli**

### CODE REVIEW

**Review by annaswims — CHANGES_REQUESTED** [review]
It looks like you didn't commit all changes in this file.  Specifically, I'm looking for the version number change.

`ActiveRecord::Schema.define(version: 2020_11_06_023750) do`

**Comment by edmangimelli** [comment]
> It looks like you didn't commit all changes in schema.rb. Specifically, I'm looking for the version number change.
> 
> `ActiveRecord::Schema.define(version: 2020_11_06_023750) do`

thanks for catching that!

File: db/migrate/20201106023750_add_birls_columns_to_form526_submissions.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+class AddBirlsColumnsToForm526Submissions < ActiveRecord::Migration[6.0]
+  def change
+    enable_extension 'pgcrypto'
+
+    add_column(
+      :form526_submissions,
+      :multiple_birls,
+      :boolean,
+      comment: '*After* a SubmitForm526 Job fails, a lookup is done to see if the veteran has multiple ' \
```
Comment (annaswims) [inline db/migrate/20201106023750_add_birls_columns_to_form526_submissions.rb:11]: ❤️ Comments!

**Review by annaswims — APPROVED** [review]
LGTM

**Review by ericboehs — APPROVED** [review]

**Pull request merged**